index.js

```diff
@@ -43,6 +43,79 @@
 //.
 //. Sanctuary is designed to work in Node.js and in ES5-compatible browsers.
 //.
+//. ## Ramda
+//.
+//. Ramda provides a number of partial functions. These necessitate the use
```

Comment:
I'm not sure if all readers will know what "partial function" means. I learned what it means not too long ago, and consider it to be functional jargon. Maybe we should consider either linking it to its meaning, or use the phrase "functions which might return `null` or `undefined`".